.github/workflows/ci.yml

```diff
@@ -246,7 +246,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image: [ngf, nginx, nginx-plus]
+        image: [ngf, nginx, plus]
```

Comment:
Why this change? To create a new cache? This won't affect the name of the image that's built though, right?

Reply:
yeah I thought it might cause problems with the cache if the name starts with the same root `nginx`, so just to be safe I changed it. The image name will remain the same.